cmd/kube-proxy/app/server.go

```diff
@@ -383,7 +386,7 @@ type ProxyServer struct {
 
 // createClients creates a kube client and an event client from the given config and masterOverride.
 // TODO remove masterOverride when CLI flags are removed.
-func createClients(config componentconfig.ClientConnectionConfiguration, masterOverride string) (clientset.Interface, v1core.EventsGetter, error) {
+func createClients(config componentconfig.ClientConnectionConfiguration, masterOverride string) (clientset.Interface, v1core.CoreV1Interface, error) {
```

Comment:
Why not just use the first return value - `clientset.Interface` - for reporting the version?